src/oss-detect-backdoor/DetectBackdoorTool.cs

```diff
@@ -49,6 +49,12 @@ public static IEnumerable<Example> Examples
             [Option('c', "use-cache", Required = false, Default = false,
                 HelpText = "do not download the package if it is already present in the destination directory.")]
             public bool UseCache { get; set; }
+
+            [Option('b', "backtracking", Required = false, HelpText = "disable using non-backtracking regex engine")]
```

Comment:
change to remove double negative? (enable backtracking)

Reply:
Changed description to "Use backtracking engine by default"